manifests/plugin/rabbitmq.pp

```diff
@@ -64,10 +65,17 @@
 
   $_manage_package = pick($manage_package, $::collectd::manage_package)
 
-  if $_manage_package {
+  if ($_manage_package) and ($provider_proxy) {
```

Comment:
can you here only set $install_options to `[{'--proxy' => $provider_proxy}]` and in the else part to `undef`?

This allows us to only define the package resource once.